src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -320,6 +320,20 @@ class MessageHelper {
       ApplicationManager& app_mngr,
       const ButtonSubscriptions& button_subscriptions);
 
+  /**
+   * @brief Creates button subscription request to hmi
+   * @param app_id - id of application for which request should be created
+   * @param button - button to be subscribed
+   * @param app_mngr - application manager
```

Comment:
missing description of function parameter

Reply:
Done in d5b3c6b